sql/2019/06_Fonts/06_46.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# 06_46: % of pages linking to a Google Fonts stylesheet
+CREATE TEMP FUNCTION preloadsGoogleFont(payload STRING)
+RETURNS BOOLEAN LANGUAGE js AS '''
+  try {
+    var $ = JSON.parse(payload);
+    var almanac = JSON.parse($._almanac);
+    return !!almanac['link-nodes'].find(e => e.rel.toLowerCase() == 'stylesheet' && e.href.startsWith('https://fonts.googleapis.com'));
```

Comment:
Don't forget about http://
```suggestion
    return !!almanac['link-nodes'].find(e => e.rel.toLowerCase() == 'stylesheet' && (e.href.startsWith('https://fonts.googleapis.com') || e.href.startsWith('http://fonts.googleapis.com')));
```

Reply:
Good catch. This actually increased the results from ~25 to ~30%.